tests/regression/test_slate_assemble_tensors.py

```diff
@@ -4,20 +4,9 @@
 from firedrake import *
 
 
-@pytest.fixture(scope='module', params=[interval, triangle, tetrahedron, quadrilateral])
+@pytest.fixture(scope='module', params=[False, True])
 def mesh(request):
-    """Generate a mesh according to the cell provided."""
-    cell = request.param
-    if cell == interval:
-        return UnitIntervalMesh(1)
-    elif cell == triangle:
-        return UnitSquareMesh(1, 1)
-    elif cell == tetrahedron:
-        return UnitCubeMesh(1, 1, 1)
-    elif cell == quadrilateral:
-        return UnitSquareMesh(1, 1, quadrilateral=True)
-    else:
-        raise ValueError("%s cell not recognized" % cell)
+    return UnitSquareMesh(2, 2, quadrilateral=request.param)
```

Comment:
Short answer: it was a time-consuming and a long-winded test. I can revert this if the time it takes to run tests isn't a factor for us.